api/src/main/java/io/opencensus/trace/unsafe/ContextUtils.java

```diff
@@ -39,7 +37,34 @@ private ContextUtils() {}
    * The {@link io.grpc.Context.Key} used to interact with {@link io.grpc.Context}.
    *
    * @since 0.5
+   * @deprecated from API since 0.21. Use {@link #withValue(Context, Span)} and {@link
+   *     #getValue(Context)} instead.
+   */
+  // TODO(songy23): make this private once gRPC migrates to use the alternative APIs.
+  @Deprecated
+  public static final Context.Key<Span> CONTEXT_SPAN_KEY =
+      Context.<Span>keyWithDefault("opencensus-trace-span-key", BlankSpan.INSTANCE);
```

Comment:
we cannot change the default for the key.

Reply:
Makes sense, fixed.